lib/elixir/lib/io/ansi.ex

```diff
@@ -46,6 +52,14 @@ defmodule IO.ANSI do
   @spec color(0..255) :: String.t
   def color(code) when code in 0..255, do: "\e[38;5;#{code}m"
 
+  @doc ~S"""
+  Converts individual R, G, B values to a single number suitable for color/1."
+
+  Valid values for each color value are in the range 0 to 5.
+  """
+  @spec rgb216(0..5, 0..5, 0..5) :: String.t
+  def rgb216(r, g, b) when valid_rgb216(r, g, b), do: 16 + (36 * r) + (6 * g) + b
```

Comment:
Yeah, that makes a lot more sense.  I could make color/3 that calls color/1, so either method is still available.  I'll make that tweak and re-submit, thanks for the input!